enterprise/ha/src/test/java/org/neo4j/ha/TestBranchedData.java

```diff
@@ -200,6 +206,13 @@ public void shouldCopyStoreFromMasterIfBranchedInLiveScenario() throws Throwable
         assertTrue( hasNode( odin, "0-0" ) );
     }
 
+    private BranchMonitor installBranchedDataMonitor( HighlyAvailableGraphDatabase odin )
+    {
+        BranchMonitor monitor = new BranchMonitor();
+        odin.getDependencyResolver().resolveDependency( Monitors.class ).addMonitorListener( monitor );
+        return monitor;
+    }
+
     private void retryOnTransactionFailure( ThrowingAction<Exception> transaction ) throws Exception
```

Comment:
Ah yes we could, that thought may have flown through my head, but was ignored before it exited!